shared/uavobjectdefinition/inssettings.xml

```diff
@@ -18,6 +18,6 @@
 		<access gcs="readwrite" flight="readwrite"/>
 		<telemetrygcs acked="true" updatemode="onchange" period="0"/>
 		<telemetryflight acked="true" updatemode="onchange" period="0"/>
-		<logging updatemode="manual" period="0"/>
+		<logging updatemode="manual" period="1"/>

```

Comment:
The use of 1 as a substitute for "always" is unfortunate here, because it advances things up to the front of the linked list where they compete with actual frequent events.